src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -135,8 +139,11 @@ internal void BindToDataSource(MirrorData mirrorData)
             }
             else if (this.IsExpanded)
             {
-                RefreshExpandedDisplay();
-                BeginViewSizeTransition(ComputeLargeContentSize());
+                RefreshExpandedDisplayAsync();
+
+                scheduler.ScheduleForExecution(
+                    new DelegateBasedAsyncTask(scheduler, 
+                        () => Dispatcher.BeginInvoke((Action)(() => BeginViewSizeTransition(ComputeLargeContentSize())))));
```

Comment:
This call to `BeginViewSizeTransition` is to be moved into `RefreshExpandedDisplayAsync` (see below for details).